recipe/meta.yaml

```
@@ -35,13 +35,13 @@ requirements:
 
 test:
   requires:
-    - pytest
+    - pytest<8
```

```suggestion
    - pytest <8
```
I'm not sure why the linter didn't complain, but we likely need a space here.